src/app/dashboard/projects/projects.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="d-flex h-100 gap-2 px-2">
+  @for (allocation of allocationsData; track allocation) {
+    <div class="d-flex flex-column p-2 rounded-3 bg-white">
+      <div class="ps-2 d-flex justify-content-between">
+        <p class="text-dark fs-5 fw-semibold">{{ allocation.project.name }}</p>
+        <div class="d-flex gap-2">
+          <div
+            class="bg-card rounded-circle project-circle d-flex justify-content-center align-items-center lh-1 text-dark">
+            {{ allocation.students.length }}
+          </div>
+          <div
+            class="rounded-circle project-circle d-flex justify-content-center align-items-center lh-1 fs-5 text-dark"
+            [matTooltip]="allocation.error.info"
+            [ngClass]="allocation.error.error ? 'bg-warn' : 'bg-card'">
+            <fa-icon
+              [icon]="allocation.error.error ? facErrorIcon : facCheckIcon"
+              [ngClass]="allocation.error.error ? 'text-white' : 'text-dark'"></fa-icon>
+          </div>
+        </div>
+      </div>
+      <app-student-preview-card [student]="emptyStudent" class="overflow-hidden h-0"></app-student-preview-card>
```

Comment:
What is the purpose of having an empty Student Preview Card?

Reply:
So that the project container keeps the same width, as the width is dependent of the student preview card